src/errors/RPCError.js

```diff
@@ -0,0 +1,22 @@
+class RPCError extends Error {
+  constructor(message, data) {
```

Comment:
Please define JSDoc. `data` supposed to be `Object`.

Reply:
added